### PR TITLE
Rm pstory2 task cohort filter

### PR DESCRIPTION
**Pull request by alecorra**

### CODE REVIEW

File: public/js/cohortFilter.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+            let row = date.parentNode
+            mismatches.push(row)
+        }
+    })
+    mismatches.forEach(function(mismatch) {
+        mismatch.classList.add('hidden')
+    })
+}
+
+function clearFilter() {
```
Comment (Liliya-V) [inline public/js/cohortFilter.js]: Dock block?

File: public/js/cohortFilter.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
+let tableRows = document.querySelectorAll('.table-row')
+let applicantsDates = document.querySelectorAll('.applicants-cohort-date')
+
+function applyCohortFilter(cohortDate) {
```
Comment (Liliya-V) [inline public/js/cohortFilter.js]: Dockblock?

File: public/js/cohortFilter.js (hunk 1 above)
Comment (alecorra) [reply public/js/cohortFilter.js]: done

File: public/js/cohortFilter.js (hunk 2 above)
Comment (alecorra) [reply public/js/cohortFilter.js]: done

**Pull request merged**